--- SYMPHONY/include/sym_win32_time.h	2012-08-09 18:39:21.000000000 -0400
+++ ../SYMPHONY-5.6.13-new/SYMPHONY/include/sym_win32_time.h	2016-04-07 16:20:16.873513700 -0400
@@ -4,7 +4,7 @@
 #if defined(_MSC_VER) || defined (__MNO_CYGWIN)
 #include <time.h>
 
-#ifndef __MINGW64__
+#if !defined(__MINGW64__) && !defined(__MINGW32__)
 struct timeval {
     long    tv_sec;         /* seconds */
     long    tv_usec;        /* and microseconds */
